python/src/deltachat/account.py

```diff
@@ -100,7 +100,7 @@ def set_config(self, name, value):
         """
         self._check_config_key(name)
         name = name.encode("utf8")
-        if name == b"addr" and self.is_configured():
+        if name == "addr" and self.is_configured():
```

Comment:
The line above does `name.encode()` which turns it into bytes and thus the comparison with bytes is correct.  Hence why the test fails on CI.

Reply:
oh yes totally overlooked that, because someone setting up a bot told me that the bot failed to run until he removed the "b" from that line, not sure was going on there

Reply:
Strange, I'm failing to think how you could end up in such a situation but with either python2 or 3.  Do you have any information on the python version?

**Maybe** it's possible that they disabled the is_configured check if they were doing this on python 3.  I think there changing it to be `name == "addr"` ensures that the `self.is_configured()` check is skipped.  **If** this theory is right it might make more sense to try and find out why they needed to do this and what effect it has on deltachat and whether it's a desired usecase to support I guess.